2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -1663,13 +1663,12 @@ A causa dei problemi incontrati, alcuni ruoli hanno visto un monte ore effettivo
 Tutti gli obiettivi sono stati raggiunti.
 
 === Problematiche
-Il gruppo ha notato in fase di retrospettiva una scarsa reattività in fase di review e verifica, che portano al rallentamento del lavoro. Le review
-sono state rallentate anche dalla presenza di numerosi merge conflicts, che il gruppo si è ritrovato a gestire per la prima volta nel progetto.
+Il gruppo ha notato in fase di retrospettiva una scarsa reattività in fase di review e verifica, che portano al rallentamento del lavoro. Le review sono state rallentate anche dalla presenza di numerosi merge conflicts, che il gruppo si è ritrovato a gestire per la prima volta nel progetto.
```

Comment:
Se mantieni inalterato il termine “fase” nella parte “in fase di retrospettiva” questo termine finisce per ripetersi subito dopo in “in fase di review”